index.html

```diff
@@ -3,70 +3,68 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Echezona's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="image/images.png" type="image/x-icon">
 </head>
 <body>
 
     <nav>
       <div class="navbar">
         <div class="logo">My Logo</div>
         <a href="#" class="menu-button">
-          <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <svg class="bar" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
             <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
         </svg>
         </a>
         <div class="navbar-links">
-          <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
+          
+          <ul id="hamburger-menu">
+            <a href="#" class="close nav-link">
+              <img src="image/1544641784.png" class="close-button">
+            </a>
+            <li class="nav-links"><a href="#portfolio">Portfolio</a></li>
+            <li class="nav-links"><a href="#about">About</a></li>
+            <li class="nav-links"><a href="#contact">Contact</a></li>
           </ul>
```

Comment:
- [ ] Great job working on your mobile menu! 💯 However, as suggested by the previous reviewer, please stick as much as possible to the given Figma design 🎨. This will showcase your ability to translate designer requirements into code!

At the moment, you are using the wrong background color as well as not covering the entire screen!

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/203548594-975c28e9-2572-4cb4-b1e7-2f53f8d39852.png) | ![imageedit_2_3767161438](https://user-images.githubusercontent.com/77069011/203548477-7b518dd9-e349-4e64-82e3-68e57ae56db2.gif) |

Please consider fixing this! 🤓 🛠️